drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -717,12 +717,14 @@ def _drop_item_assessment(self, item_attempt):
 
         item = self._get_item_definition(item_attempt['val'])
         is_correct = self._is_attempt_correct(item_attempt)
+        str_item_id = str(item['id'])
         if item_attempt['zone'] is None:
-            del self.item_state[str(item['id'])]
```

Comment:
I think if we replace this line with this statement we don't need to do anything else in this PR
```
self.item_state.pop(str(item['id']), None)
```

Reply:
Makes sense. 👍